Dockerfiles/manifests/openshift/scc.yaml

```diff
@@ -43,6 +43,8 @@ fsGroup:
   type: MustRunAs
 readOnlyRootFilesystem: false
 runAsUser:
+# With some versions of Openshift, you may need to run the agent as a root user to access the Docker socket. 
+# In that case, use `type: RunAsAny` instead
   type: MustRunAsRange
```

Comment:
Another solution it to create a `group` that have access to the docker socket. then add this group in the`supplementalGroups`.
doc: https://docs.openshift.com/enterprise/3.1/install_config/persistent_storage/pod_security_context.html#supplemental-groups

Reply:
Yes, this seems to be a better solution.